download-basket/src/main/java/downloadbasket/actions/GetFeatureForCropping.java

```diff
@@ -67,6 +72,9 @@ public void handleAction(final ActionParameters params) throws ActionException {
 			try {
 
 				HttpURLConnection con = IOHelper.getConnection(wmsUrl);
+				if(hasAuthorization()){
+					IOHelper.setupBasicAuth(con,croppingUsername, croppingPassword);
+				}
```

Comment:
You just pass user/pw to IOHelper.getConnection(url, user, pw). It won't setup basic auth if username is null or empty.